docs/src/ComboBox.doc.js

```diff
@@ -0,0 +1,732 @@
+// @flow strict
+import type { Node } from 'react';
+import PropTable from './components/PropTable.js';
+import PageHeader from './components/PageHeader.js';
+import MainSection from './components/MainSection.js';
+
+const cards: Array<Node> = [];
+const card = (c) => cards.push(c);
+
+card(
+  <PageHeader
+    name="ComboBox"
+    description="ComboBox is the combination of a Textfield and an associated Dropdown to let the user filter a list when selecting. ComboBox allows to type the selection, type part of the selection and narrow the results, and select an option from the list."
+    defaultCode={`
+function ComboBoxExample(props) {
+  const PRONOUNS = [
+    'ey / em',
+    'he / him',
+    'ne / nem',
+    'she / her',
+    'they / them',
+    've / ver',
+    'xe / xem',
+    'xie / xem',
+    'zie / zem',
+  ];
+
+  const options = PRONOUNS.map((pronoun, index) => ({ label: pronoun, value: 'value'+index }));
+
+  return (
+    <ComboBox
+      accessibilityClearButtonLabel="Clear the current value"
+      accessibilityShowButtonLabel="Show a list of pronoun values for this field"
```

Comment:
Same comment here, since the tab index is -1, I think this prop isn't needed, and we could instead use role="presentation" instead of button when it's the arrow-down

Reply:
DONE

Reply:
It's a TapArea under the hood, we don't support `accessibilityRole` in TapArea. Leaving as is for now.

Reply:
DONE